src/modules/onboarding/introduction/SliderContent/index.jsx

```diff
@@ -0,0 +1,20 @@
+import { Typography } from '@mui/material';
+import { Title1 } from '../../../../shared-components/Title1';
+import './styles.css';
+
+export const SliderContent = ({ activeIndex, sliderData }) => {
+  return (
+    <>
+      {sliderData.map((slide, index) => (
+        <div
+          key={index}
+          className={index === activeIndex ? 'slides active' : 'inactive'}
+        >
+          <img src={slide.image} alt='' />
+          <Title1>{slide.title}</Title1>
+          <Typography variant='body1'>{slide.description}</Typography>
```

Comment:
This should be a share, I think Regular1 in the design https://www.figma.com/file/6KSlLFezGkD8uZXhuTsx3j/Montra---Expense-Tracker-UI-Kit-(Community)?type=design&node-id=611-3750&mode=design&t=UkLZq6wPCcYQEqoC-0